src/StorageSync/StorageSync/help/Register-AzStorageSyncServer.md

```diff
@@ -0,0 +1,152 @@
+---
+external help file: Microsoft.Azure.Commands.StorageSync.dll-Help.xml
+Module Name: Az.StorageSync
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.storagesync/register-Azstoragesyncserver
+schema: 2.0.0
+---
+
+# Register-AzStorageSyncServer
+
+## SYNOPSIS
+This command will use to register sync server.
+
+## SYNTAX
+
+### ObjectParameterSet (Default)
+```
+Register-AzStorageSyncServer [-ParentObject] <PSStorageSyncService> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### StringParameterSet
+```
+Register-AzStorageSyncServer [-ResourceGroupName] <String> [-StorageSyncServiceName] <String> [-AsJob]
```

Comment:
StorageSyncServiceName -> Name

Reply:
StorageSyncServer is different than StorageSyncService

---
In reply to: [255696091](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255696091) [](ancestors = 255696091)